subhub/tests/unit/test_payment_calls.py

```diff
@@ -246,6 +246,82 @@ def test_cancel_subscription_with_valid_data(app, create_subscription_for_proces
     g.subhub_account.remove_from_db("process_test")
 
 
+def test_cancel_subscription_with_valid_data_multiple_subscriptions_remove_first():
+    """
+    GIVEN a user with multiple subscriptions
+    WHEN the first subscription is cancelled
+    THEN the specified subscription is cancelled
+    """
+    uid = uuid.uuid4()
+    subscription1, code1 = payments.subscribe_to_plan(
+        "valid_customer",
+        {
+            "pmt_token": "tok_visa",
+            "plan_id": "plan_EtMcOlFMNWW4nd",
+            "email": f"valid@{uid}customer.com",
+            "orig_system": "Test_system",
+        },
+    )
+    subscription2, code2 = payments.subscribe_to_plan(
+        "valid_customer",
+        {
+            "pmt_token": "tok_visa",
+            "plan_id": "plan_F4G9jB3x5i6Dpj",
+            "email": f"valid@{uid}customer.com",
+            "orig_system": "Test_system",
+        },
+    )
+
+    (cancelled, code) = payments.cancel_subscription(
+        "valid_customer", subscription1["subscriptions"][0]["subscription_id"]
+    )
+    assert cancelled["message"] == "Subscription cancellation successful"
+    assert 201 == code
+
+    g.subhub_account.remove_from_db("valid_customer")
+    (cancelled, code) = payments.cancel_subscription(
+        "valid_customer", subscription2["subscriptions"][0]["subscription_id"]
+    )
```

Comment:
Would it be useful to verify what the code returned here is?

Reply:
Yes it would be useful to verify that it also had been removed. I will add the check.